src/models/userDao.js

```diff
@@ -0,0 +1,40 @@
+const { appDataSource } = require("./appDataSource");
+
+const getUserInfo = async (socialId, socialTypeId) => {
+  const userInfo = await appDataSource.query(
+    `SELECT
+      id
+    FROM
+      users
+    WHERE
+      social_id = ? AND social_type_id = ?`,
+    [socialId, socialTypeId]
+  );
+
+  return userInfo;
+};
```

Comment:
service layer code에서 배열의 대괄호를 제거하는것 보다, getUserInfo라면 특정 유저의 정보, 즉 하나의 오브젝트만 반환되는것이 올바른 return 값이기 때문에 여기서 배열을 제거해주는 것이 나을것 같습니다.
```suggestion
const getUserInfo = async (socialId, socialTypeId) => {
  const [userInfo] = await appDataSource.query(
    `SELECT
      id
    FROM
      users
    WHERE
      social_id = ? AND social_type_id = ?`,
    [socialId, socialTypeId]
  );

  return userInfo;
};
```